src/redux/backend-session.ts

```diff
@@ -222,6 +224,19 @@ const registerMux = (language: Language, dclId: string) => {
   });
 };
 
+const selectImageTag = (language: Language, version: string) => {
+  if(language == "python"){
+    const python_versions = ["3.5", "3.6", "3.7"];
+    const default_version = "3.5";
+    if(version && python_versions.includes(version)) {
+      return "dcl-python-"+version+":latest";
+    }
+    console.log("Python version not found. Using default Python" + default_version + ". Versions available are: " + python_versions);
+    return "dcl-python-"+default_version+":latest";
+  }
+  return "";
```

Comment:
I'm surprised by this return, shouldn't it be `` `dcl-python-${version}:latest` ``?